src/gui/MainWindow.cc

```diff
@@ -2195,6 +2199,55 @@ void MainWindow::sendToOctoPrint()
 #endif // ifdef ENABLE_3D_PRINTING
 }
 
+void MainWindow::sendToLocalSlicer(void)
+{
+#ifdef ENABLE_3D_PRINTING
+  const QString slicer = QString::fromStdString(Settings::Settings::localSlicerExecutable.value());
+
+ const QString fileFormat = QString::fromStdString(Settings::Settings::octoPrintFileFormat.value());
+  FileFormat exportFileFormat{FileFormat::STL};
+  if (fileFormat == "OBJ") {
+    exportFileFormat = FileFormat::OBJ;
+  } else if (fileFormat == "OFF") {
+    exportFileFormat = FileFormat::OFF;
+  } else if (fileFormat == "ASCIISTL") {
+    exportFileFormat = FileFormat::ASCIISTL;
+  } else if (fileFormat == "AMF") {
+    exportFileFormat = FileFormat::AMF;
+  } else if (fileFormat == "3MF") {
+    exportFileFormat = FileFormat::_3MF;
+  } else {
+    exportFileFormat = FileFormat::STL;
+  }
+
+  std::shared_ptr<QTemporaryFile> exportFile = std::shared_ptr<QTemporaryFile>(new QTemporaryFile({QDir::temp().filePath("OpenSCAD.XXXXXX."+fileFormat.toLower())}));
+  exportFile->setAutoRemove(false);
+  if (!exportFile->open()) {
+    LOG("Could not open temporary file.");
+    return;
+  }
+  const QString exportFileName = exportFile->fileName();
+  exportFile->close();
+
+  QString userFileName;
+  if (activeEditor->filepath.isEmpty()) {
+    userFileName = exportFileName;
+  } else {
+    QFileInfo fileInfo{activeEditor->filepath};
+    userFileName = fileInfo.baseName() + fileFormat.toLower();
+  }
+
+  ExportInfo exportInfo = createExportInfo(exportFileFormat, exportFileName, activeEditor->filepath);
+  exportFileByName(this->root_geom, exportInfo);
+
+  QProcess process(this);
+  if(!process.startDetached(slicer, {exportFileName})) {
+    LOG(message_group::Error, "Could not start Slicer.  Is it installed?");
```

Comment:
This error message is not very informative. Would it be possible to provide more info why it failed? e.g. by using `setStandardErrorFile()` to extract error output?

Reply:
I kept that in using `readAll()` with merged channels, but I don't think it will ever report anything. This will likely happen only in the case where the call actually succeeds and then later collects the output.
Unfortunately the dedicated error message for `startDetached()` failure does not give a very useful error message for the case of a wrong path.